tcms/testplans/templates/testplans/mutable.html

```diff
@@ -24,6 +32,18 @@
                 </div>
             </div>
 
+        {% if test_plan %}
+            <div class="form-group">
+                <label for="id_active" class="col-md1 col-lg-1"> {% trans "Active" %}</label>
+                <div class="col-md-1 col-lg-1"> {{ form.is_active }} </div>
+            
+                <div>
+                    <label class="col-md-1 col-lg-1"> {% trans "Author" %}</label>
+                    <span class="col-md-2 col-lg-2"><a href="{% url "tcms-profile" test_plan.author.username %}">{{ test_plan.author.username }}</a></span>
+                </div>
+            </div>
+        {% endif %}
```

Comment:
This entire block should be placed after Reference link and before the document. This will minimize the disruption of the UI flow. Try opening the new and edit pages side by side and switch tabs between the two.

Reply:
Since the author field will be removed, do I still have to place the Active button after Reference link? And do you mean in the same column (shrink the ref link), or below the ref link?